### PR TITLE
simple counter solution for long query

### PR DESCRIPTION
**Pull request by Nuanda**
Makes the query both fast and simple. Provided the data submission process would be infrequent, perhaps having a simple counter is all right.

### CODE REVIEW

File: lib/tasks/app.rake [hunk 1]
```diff
@@ -28,12 +30,16 @@ namespace :app do
   def restore_cropstore_dump
     db_config = Rails.application.config_for(:database)
     env = { "PGUSER" => db_config["username"], "PGPASSWORD" => db_config["password"] }
-    cmd = "pg_restore -O -d #{db_config["database"]} db/cropstore_web_20150403_1530.dump"
+    cmd = "pg_restore -O -d #{db_config["database"]} db/cropstore_dev_20150403_1530.dump"
```
Comment (nowakowski) [inline lib/tasks/app.rake]: Please undo this change; I've already fixed the filename in master.

File: app/models/plant_population.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class PlantPopulation < ActiveRecord::Base
-  include Elasticsearch::Model
-  include Elasticsearch::Model::Callbacks
+  # include Elasticsearch::Model
+  # include Elasticsearch::Model::Callbacks
```
Comment (nowakowski) [inline app/models/plant_population.rb]: Why are these commented out?

Comment (Nuanda) [inline app/models/plant_population.rb]: fixed

File: lib/tasks/app.rake (hunk 1 above)
Comment (Nuanda) [inline lib/tasks/app.rake]: So it should merge properly.

Comment (nowakowski) [inline lib/tasks/app.rake]: I renamed the file itself, not the reference in `app.rake`. It won't merge properly.

**Comment by nowakowski** [comment]
:+1:
